generators/app/templates/.gitignore

```diff
@@ -18,39 +18,32 @@ Thumbs.db
 
 # Compiled Files
 # ---------------------------
-/dist/<% if (platform == 'craft') { %>
+/dist/
 
 # Craft
 # ---------------------------
 /craft/plugins/*
 /craft/rev-manifest.json
-/craft/storage/
+/craft/storage/*
 
 # Craft Public
 # ---------------------------
-/public/.htaccess
-/public/assets/
-/public/assets/**/_*
-/public/images/
-/public/scripts/
-/public/styles/
-/public/uploads/
-/public/web.config<% } %>
+/public/*
+!/public/index.php
+!/public/*.txt
+!/public/*.png
+!/public/*.ico
 
 # Package Managers
 # ---------------------------
-/.bundle
 /vendor/
-bower_components
-node_modules
-npm-debug.log
+/node_modules/
```

Comment:
I buy that argument for `bundle` and to some extent for `/bower_components` (even though this repo does support Bower, I think we should remove that`. `nom-debug.log` though _will_ get generated eventually by this generator if an npm install fails.